files/grest/rpc/epoch/epoch_info.sql

```diff
@@ -1,4 +1,7 @@
-CREATE FUNCTION grest.epoch_info (_epoch_no numeric DEFAULT NULL)
+
+DROP FUNCTION IF EXISTS grest.epoch_info (_epoch_no numeric);
```

Comment:
can delete this line (as it adds a backlog delete to be cleared for an item that will not exist past next release), running setup-grest.sh -q will drop all functions